tests/testdata/minimal/debug_out.toml

```diff
@@ -24,8 +24,13 @@ Promiscuous = false
 [[NodeGroups]]
 Prefix = 'users'
 NoNodes = 7
+PredefinedPosition = false
 NodesType = 1
 
+[NodeGroups.Position]
```

Comment:
Please try to stick to the style used in the other examples

Reply:
one cannot change the output data of the debug_out.toml, which is used to test the generate functions